src/core/server/saved_objects/migrations/core/transform_saved_object_document_error.test.ts

```diff
@@ -11,50 +11,30 @@ describe('TransformSavedObjectDocumentError', () => {
   it('is a special error', () => {
     const originalError = new Error('Dang diggity!');
     const err = new TransformSavedObjectDocumentError(
-      'id',
-      'type',
-      'namespace',
       'failedTransform',
       'failedDoc',
       originalError
     );
     expect(err).toBeInstanceOf(TransformSavedObjectDocumentError);
-    expect(err.id).toEqual('id');
-    expect(err.namespace).toEqual('namespace');
     expect(err.stack).not.toBeNull();
+    expect(err.originalError).toBe(originalError);
   });
   it('constructs an special error message', () => {
     const originalError = new Error('Dang diggity!');
     const err = new TransformSavedObjectDocumentError(
-      'id',
-      'type',
-      'namespace',
       'failedTransform',
       'failedDoc',
       originalError
     );
-    expect(err.message).toMatchInlineSnapshot(
-      `
-      "Failed to transform document id. Transform: failedTransform
-      Doc: failedDoc"
-    `
-    );
+    expect(err.message).toMatchInlineSnapshot(`"Dang diggity!"`);
```

Comment:
We use the [original transformation error](https://github.com/elastic/kibana/pull/100297/files#diff-fa8464290b3033bbb528b3e42032a13fbfde59781fc79a8b0c624114f9e82589R23) message and don't construct one anymore.